pkg/controllers/v1alpha2/mpi_job_controller.go

```diff
@@ -522,13 +525,13 @@ func (c *MPIJobController) getLauncherJob(mpiJob *kubeflow.MPIJob) (*batchv1.Job
 	return launcher, nil
 }
 
-// getOrCreatePDB will create a PDB for gang scheduling by kube-batch.
-func (c *MPIJobController) getOrCreatePDB(mpiJob *kubeflow.MPIJob, minAvailableWorkerReplicas int) (*policyv1beta1.PodDisruptionBudget, error) {
+// getOrCreatePodGroups will create a PodGroup for gang scheduling by kube-batch.
+func (c *MPIJobController) getOrCreatePodGroups(mpiJob *kubeflow.MPIJob, minAvailableWorkerReplicas int32) (*podgroupv1alpha2.PodGroup, error) {
```

Comment:
Now the count of worker will not be set to 0 when the training work completed. So only has the CreatePodGroup func, will add DeletePodGroup alone with set worker to 0 in follow-up PRs